src/Fixer/FunctionNotation/ReturnTypeDeclarationFixer.php

```diff
@@ -53,8 +103,31 @@ public function fix(\SplFileInfo $file, Tokens $tokens)
     /**
      * {@inheritdoc}
      */
-    protected function getDescription()
+    public function getDefinition()
     {
-        return 'There should be no space before colon and one space after it in return type declaration.';
+        $versionSpecification = new VersionSpecification(70000);
+
+        return new FixerDefinition(
+            'There should be one or no space before colon, and one space after it in return type declarations, according to configuration.',
+            array(
+                new VersionSpecificCodeSample(
+                    "<?php\nfunction foo() : string {};",
+                    $versionSpecification
```

Comment:
sure, note is ok.
if you want to do it in separated pr im fine with that. just be aware that i will wait with merging this pr until new one will be merged to not provide php5 users bad experience